rct229/rulesets/ashrae9012019/section19/section19rule13.py

```diff
@@ -0,0 +1,255 @@
+            [
+                baseline_system_type_compare(system_type, applicable_sys_type, False)
+                for system_type in baseline_system_types_dict.keys()
+                for applicable_sys_type in APPLICABLE_SYS_TYPES
+            ]
+        )
+
+    class HVACRule(RuleDefinitionBase):
+        def __init__(self):
+            super(Section19Rule13.HVACRule, self).__init__(
+                rmrs_used=UserBaselineProposedVals(False, True, True),
+                required_fields={
+                    "$": ["fan_system"],
+                    "fan_system": ["minimum_outdoor_airflow"],
+                },
+            )
+
+        def get_calc_vals(self, context, data=None):
+            hvac_b = context.baseline
+            hvac_id_b = hvac_b["id"]
+            zone_info = data["zone_info"][hvac_id_b]
+            all_design_setpoints_delta_Ts_are_per_reqs_b = zone_info[
+                "all_design_setpoints_delta_Ts_are_per_reqs"
+            ]
+            are_all_hvac_sys_fan_objs_autosized_b = zone_info[
+                "are_all_hvac_sys_fan_objs_autosized"
+            ]
+
+            fan_sys_b = hvac_b["fan_system"]
+            supply_fans_airflow_b = (
+                get_fan_system_object_supply_return_exhaust_relief_total_power_flow(
+                    fan_sys_b
+                )
+            )
+            fan_minimum_outdoor_airflow_b = fan_sys_b["minimum_outdoor_airflow"]
+
+            return {
+                "all_design_setpoints_delta_Ts_are_per_reqs_b": all_design_setpoints_delta_Ts_are_per_reqs_b,
+                "are_all_hvac_sys_fan_objs_autosized_b": are_all_hvac_sys_fan_objs_autosized_b,
+                "supply_fans_airflow_b": supply_fans_airflow_b,
+                "fan_minimum_outdoor_airflow_b": fan_minimum_outdoor_airflow_b,
+            }
+
+        def manual_check_required(self, context, calc_vals=None, data=None):
+            hvac_b = context.baseline
+            hvac_id_b = hvac_b["id"]
+            zone_info = data["zone_info"][hvac_id_b]
+            supply_fans_airflow_b = calc_vals["supply_fans_airflow_b"]
+
+            all_design_setpoints_delta_Ts_are_per_reqs_b = calc_vals[
+                "all_design_setpoints_delta_Ts_are_per_reqs_b"
+            ]
+
+            return not all_design_setpoints_delta_Ts_are_per_reqs_b and std_equal(
+                zone_info["proposed_supply_airflow"], supply_fans_airflow_b
+            )
+
+        def get_manual_check_required_msg(self, context, calc_vals=None, data=None):
+            hvac_b = context.baseline
+            hvac_id_b = hvac_b["id"]
+
+            return f"{hvac_id_b} was not modeled based on a supply-air-to-room temperature set-point difference of 20°F (or 17°F, if lab). The baseline and proposed supply cfm was modeled identically at <insert proposed_supply_airflow> CFM. Manual review is required to determine if the airflow rate was modeled to comply with applicable codes or accreditation standards. If not, fail."
+
```

Comment:
You will need to insert an airflow value here.
Add the variable to CalcQ and then call calcq_to_str with a unit system to convert the value.
You can remove the CFM.

Reply:
Good point! Addressed.